pom.xml

```diff
@@ -145,6 +145,11 @@
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>structs</artifactId>
```

Comment:
Yes, it's needed for `@Symbol`. See the jenkinsci-dev mailing list for details: https://groups.google.com/forum/#!msg/jenkinsci-dev/6ijgyOFvZbg/MDIOUKm0BAAJ